opentelemetry/README.md

```diff
@@ -0,0 +1,50 @@
+---
+title: OpenTelemetry
+---
+
+This Plugin is intended for ingesting, generating, collecting, and exporting telemetry data (metrics, logs, and traces). 
+
+Components included in this Plugin:
+
+- [Collector](https://github.com/open-telemetry/opentelemetry-collector)
+- [Filelog Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver)
+- [OpenSearch Exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/opensearchexporter)
+- ServiceMonitor to scrape logs statically into prometheus infra-frontend
```

Comment:
Is the ServiceMonitor a core component of the plugin? If yes or you want to mention it, the Prometheus part should be configurable. Are those just performance metrics or really the logs processed?
```suggestion
- ServiceMonitor to ingest the metrics of the OpenTelemetry Collector in Prometheus 
```

Reply:
You are right. I will mention it here and the detailed configuration has to be implemented in the upcoming sprints #325